components/URLPrompt.tsx

```diff
@@ -14,7 +14,7 @@ import { SelectionInfo } from "interfaces/SelectionInfo";
 import { transformLink } from "utils/transformLink";
 
 const getInputVariant = (url: string) => {
-  if (url.length === 0) return "default";
+  if (url.length === 0) return null;
```

Comment:
Let's stick to `if (url.length === 0) return undefined;` as this is the value of an optional prop when not passing a value.